README.md

```diff
@@ -1,11 +1,9 @@
-<p align="center"><img width="320px" align="center" style="width:320px" src="https://raw.githubusercontent.com/ecamp/ecamp3/devel/logo.svg"/></p><br/>
-<p align="center">
-  <a href="https://travis-ci.com/ecamp/ecamp3"><img src="https://travis-ci.com/ecamp/ecamp3.svg?branch=devel" alt="Build Status"></a>
-  <a href="https://coveralls.io/github/ecamp/ecamp3?branch=devel"><img src="https://coveralls.io/repos/github/ecamp/ecamp3/badge.svg?branch=devel" alt="Coverage Status"></a>
-  <a href="https://github.com/ecamp/ecamp3/blob/devel/LICENSE"><img src="https://badgen.net/github/license/ecamp/ecamp3" alt="License"></a>
-</p>
+# ![Logo](frontend/public/logo.svg) eCamp v3
+[![Build Status](https://travis-ci.com/ecamp/ecamp3.svg?branch=devel)](https://travis-ci.com/ecamp/ecamp3)
+[![Coverage Status](https://coveralls.io/repos/github/ecamp/ecamp3/badge.svg?branch=devel)](https://coveralls.io/github/ecamp/ecamp3?branch=devel)
+[![License](https://badgen.net/github/license/ecamp/ecamp3)](https://github.com/ecamp/ecamp3/blob/devel/LICENSE)
```

Comment:
Those were html tags for a reason... https://github.com/ecamp/ecamp3/blob/8ee1f9632660288a80b773b63a68f3455a25663b/README.md doesn't look as nice anymore ;)